docs/content/development/extensions-core/kinesis-ingestion.md

```diff
@@ -390,4 +413,15 @@ requires the user to manually provide the Kinesis Client Library on the classpat
 compatible with Apache projects.
 
 To enable this feature, add the `amazon-kinesis-client` (tested on version `1.9.2`) jar file ([link](https://mvnrepository.com/artifact/com.amazonaws/amazon-kinesis-client/1.9.2)) under `dist/druid/extensions/druid-kinesis-indexing-service/`.
-Then when submitting a supervisor-spec, set `deaggregate` to true.
\ No newline at end of file
+Then when submitting a supervisor-spec, set `deaggregate` to true.
+
+## Configuration Properties
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|druid.supervisor.stream.healthinessThreshold|The number of successful iterations before the supervisor flips from an UNHEALTHY to a RUNNING state|3|
```

Comment:
Not necessarily - it could also have been in a `LOST_CONTACT_WITH_STREAM` or `UNABLE_TO_CONNECT_TO_STREAM` which are also the 'unhealthy' states. Maybe it shouldn't be capitalized to avoid confusion?

Reply:
Similarly with the rest of the comments about `UNHEALTHY` -> `UNHEALTHY_SUPERVISOR`.